README.md

```diff
@@ -1 +1,59 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+In this project, it includes building a simple HTML LeaderBoard tasks using webpack and served by a webpack dev server.
+
+# Built with
+
+<ul>
+<li>HTML</li>
+<li>CSS</li>
+<li>JavaScript</li>
+  <li>Webpack</li>
+</ul>
+
+# Prerequisites
+
+<ul>
+<li>VSCode</li>
+<li>Git</li>
+<li>Browser With Developer Tools</li>
+</ul>
+
+# Project Setup
+
+1. Setup the project by creating a new directory and initializing it with npm using the official webpack documentation to set up webpack.
+2. Add the HTML to index.html in src folder.
+3. Worked on styling the website using style.css.
+4. Created modules for the project and imported them to index.js
```

Comment:
- [ ] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.